File: cpp/fib.cpp
#include <iostream>

using namespace std;

int fib(int n) {
    if (n <= 1) {
        return n;
    }

    return fib(n - 1) + fib(n - 2);
}

int main() {
    int value;
	cout << "Enter the number of the fibonacci to calculate: ";
    cin >> value;
    for (int i = 0; i <= value; i++) {
		cout << "The fibonacci " << i << " is " << fib(i) << endl;
    }
    return 0;
}

File: python/fib.py
'''
Calculates numbers of the Fibonacci sequence.
'''


def fib(number):
    '''
    Calculates numbers of the Fibonacci sequence.
    '''
    if number <= 1:
        return number
    return fib(number - 1) + fib(number - 2)


def main():
    '''
    Main entrypoint to the program.
    '''
    print('Enter the number of the fibonacci to calculate')
    data = int(input())
    for number in range(0, data):
        print('Fibonacci of ', number, 'is', fib(number))


if __name__ == "__main__":
    main()
